pkg/agent/openflow/client_test.go

```diff
@@ -2570,14 +2664,26 @@ func Test_client_RegisterPacketInHandler(t *testing.T) {
 }
 
 func Test_client_ReplayFlows(t *testing.T) {
+	testClientReplayFlows(t, false)
+	if !runtime.IsWindowsPlatform() {
+		testClientReplayFlows(t, true)
```

Comment:
could you use `OVSMetersAreSupported()` here instead of `runtime.IsWindowsPlatform()`?

Reply:
Changed.